components/ShoppingList.js

```diff
@@ -1,20 +1,22 @@
 import ShoppingItem from "./ShoppingItem";
 
-export default function ShoppingList({shoppingItemData}){
-    return(
-        <>
-        <ul style = { {listStyle:"none", padding: 0 }}> 
-            {shoppingItemData.map((item) => (
-                <li key={item.id}>
-                    <ShoppingItem cardImage = {item.imageUrl}
-                    cardTitle = {item.name} cardQuantity = {item.quantity} cardCategory = {item.category}
-                    />
-                </li>
-            ))
-            
-            
-            }
-        </ul>
-        </>
-    ) 
-}
\ No newline at end of file
+export default function ShoppingList({ shoppingItemData }) {
+  if (!shoppingItemData?.length) return <p>No items found.</p>;
```

Comment:
please change to the following for clarity:
```suggestion
  if (shoppingItemData?.length === 0) return <p>No items found.</p>;
```